markdown/dev/howtos/editors/frontmatter/about/en.md

```diff
@@ -6,11 +6,16 @@ order: 10
 Frontmatter is a way to add metadata to Markdown documents.
 
 Frontmatter sits at the top of the file (it's matter that's at the front) and is
-surrounded by lines with three dashes on them:
+surrounded by lines with three dashes on them. It contains several keys with a value. 
+
+The ``title:`` key is required on every page, it holds the title of the page. 
```

Comment:
One one backtick, and no colon please:

```md
The `title` key
```

The colon is not part of the key, it's syntax.

Reply:
Done!